index.html

```diff
@@ -1151,6 +1176,160 @@ <h3>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Icons
+      </h2>
+      <p>
+        Icons can be used by the user agent in multiple places, such as on a
+        home-screen, in an app store, or even on top of notifications.
+      </p>
+      <p>
+        For this reason, icons can be declared with different sizes and for
```

Comment:
I don't know if this badge example is needed - it's covered in the definition of the "purpose" member.

Reply:
ok